### ISSUE TITLE
Add error handling for API calls

### ISSUE THREAD

**Issue by johnlaine1** [johnlaine1/udacity-fend-health-tracker#11]

**Comment by johnlaine1** [comment]
Set up for Nutritionix, done.
Need to set up for firebase.

**Comment by johnlaine1** [comment]
Having problems setting this up for firebase. Using the same method as for nutritionix but it does not seem to be working.
If I create the error in the same way for Nutritionix (by altering the base url) I get the same error with Nutritionix. It seems as though it's not recognizing that I have an error callback.
I tried:
inlining an error callback function

**Comment by johnlaine1** [comment]
Backbonefire autosync is set to true, let's try setting it to false
That does not seem to work, console is showing repeated errors like the system keeps trying to connect.

I discovered that in the default, with autoSync on, I do not need to call fetch, it will actually be ignored because of firebase. So I am going to have to look into firebase for some type of error handling.

**Comment by johnlaine1** [comment]
I am unable to figure out how to do this. I can't find any info from firebase.

**Comment by johnlaine1** [comment]
what if I use some timer to show a message after 3 seconds if ...

I ended up using a setTimeout / clearTimeout solution.
Using setTimeout, I set a function to fire 5 seconds after the initial page render which would inform the user that there had been an error. I then created a call to clearTimeout in the 'addOne' method which is responsible for rendering items to the log. If the code gets to this point, it is because there was a successful connection so the error function is no longer needed.
